pkg/sql/lex/BUILD.bazel

```diff
@@ -6,6 +6,7 @@ go_library(
         "encode.go",
         "experimental_keywords.go",
         "keywords.go",  # keep
+        "reserved_keywords.go",
```

Comment:
i think this needs to be rebased

Reply:
Yes, you were right, I just rebase to the last commit in master:
e533f1939c55c34470e1c09958fe9f8205b63805

(at least by this time)

Reply:
This diff should go away, I think. This file was moved to pkg/sql/lexbase. You might have a leftover generated file from an earlier commit. You should be able to `git clean -fx` it away.

Reply:
You were right, seemed to be an issue some garbage code that was there but I already fixed it.

Thank you for the git clean command as well